src/plugins/intel_gna/src/backend/make_pwl.cpp

```diff
@@ -50,7 +51,7 @@ static void insert_extra_pwl_segments(std::vector<gna_pwl_segment_t>& gna_pwl,
 
         if (y_value > static_cast<float>(INT16_MAX) || y_value < static_cast<float>(INT16_MIN)) {
             float x_value = ((static_cast<float>(y_max) - yBase) * scale) / slope + xBase;
-            extra_segment.xBase = FLOAT_TO_INT32(x_value) & XBASEMASK;
+            extra_segment.xBase = FloatToInt32(x_value) & XBASEMASK;
```

Comment:
This is interesting. Why we need to substitute FLOAT_TO_INT... macro by functions? It looks like nothing changed in the source code.

Reply:
You are right change is not significant.
In this case macros does not give us any benefit. Function will enforce type checking. Function is inline so rather it should not impact performance.